src/server/game/World/World.cpp

```diff
@@ -793,6 +793,8 @@ void World::LoadConfigSettings(bool reload)
     m_int_configs[CONFIG_CHARTER_COST_ARENA_3v3] = sConfigMgr->GetIntDefault("ArenaTeam.CharterCost.3v3", 1200000);
     m_int_configs[CONFIG_CHARTER_COST_ARENA_5v5] = sConfigMgr->GetIntDefault("ArenaTeam.CharterCost.5v5", 2000000);
 
+    m_int_configs[CONFIG_MAX_WHO_LIST_RETURN] = sConfigMgr->GetIntDefault("MaxWhoListReturns", 49);
```

Comment:
why do not put 50 as default? like before?

Reply:
according to this comment, it always defaulted 49 because it is the most stable
```cpp
        // 49 is maximum player count sent to client - can be overridden
        // through config, but is unstable
```

Reply:
Interesting. Does anyone know how exactly would it be "unstable" with a number higher than 49?

Reply:
i think the client only lists 49 in /who